src/workerd/util/BUILD.bazel

```diff
@@ -321,3 +321,15 @@ kj_test(
         ":uuid",
     ],
 )
+
+kj_test(
```

Comment:
Perhaps add a tag called requires-perfetto instead to avoid running this when perfetto is disabled – there's no use in building and running a no-op test in that case.